types/stripe-js/payment-intents.d.ts

```diff
@@ -1432,6 +1432,13 @@ export interface ConfirmPaymentData extends PaymentIntentConfirmParams {
     allow_redisplay?: 'always' | 'limited' | 'unspecified';
   };
 
+  /**
+   * Optional `id` of an existing [ConfirmationToken](https://stripe.com/docs/api/confirmation_tokens).
+   *
+   * @docs https://stripe.com/docs/js/payment_intents/confirm_payment#confirm_payment_intent-options-confirmParams-confirmation_token
+   */
+  confirmation_token?: string;
+
```

Comment:
is there a way to test this?

Reply:
there are tests [here](https://github.com/stripe/stripe-js/blob/ed142f58aea1d2bbb79afca5970e3a8341b8bdd8/tests/types/src/valid.ts#L2809) but we don't typically include a test for an optional string. what did you have in mind?